Sources/Plasma/PubUtilLib/plDrawable/plDynaTorpedoMgr.cpp

```diff
@@ -56,13 +56,9 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "pnEncryption/plRandom.h"
 
-static const uint32_t kNumPrintIDs = 0;
-
-static plRandom sRand;
-
 plDynaTorpedoMgr::plDynaTorpedoMgr()
 {
-    fPartIDs.SetCount(kNumPrintIDs);
+    fPartIDs.SetCount(0);
```

Comment:
What's the advantage of using a literal zero here instead of a constant?  More specifically, if this should be empty, why not remove this line entirely since the default state of an `hsTArray` should be empty.